test/TestImportIncludeRole.py

```diff
@@ -1,8 +1,6 @@
 import pytest
-
```

Comment:
The problem here is the same as in other places: isort thinks that ansiblelint is third-party and puts it together with pytest.